src/Sulu/Component/Content/Document/Structure/PropertyValue.php

```diff
@@ -87,7 +87,7 @@ public function offsetSet($offset, $value)
             return;
         }
 
-        $this->value[$offset] = array_merge($this->value[$offset], $value);
+        $this->value[$offset] = $value;
```

Comment:
this error happens in my opinion only in preview which could lead in a non functional preview . but shouldnt be "blocking" for the user